src/components/dev-hub/events.js

```diff
@@ -75,18 +90,7 @@ const StyledEvent = styled(Link)`
     padding: ${size.medium};
     text-decoration: none;
     transition: background-color ${animationSpeed.medium};
-
-    &:focus,
-    &:hover {
-        color: ${({ theme }) => theme.devWhite};
-        background: ${({ theme }) => theme.greyDarkTwo};
-        div[data-name='event-date'] {
-            background: ${({ theme }) => theme.greenTeal};
-        }
-        p[data-name='event-location'] {
-            color: ${({ theme }) => theme.devWhite};
-        }
-    }
+    ${({ theme }) => hoverStyles(theme)}
```

Comment:
Nice refactor! This is likely where that weirdness with the hover state is coming in